### ISSUE TITLE
Selectboxit jquery plugin is not accessible for screen reader user

### ISSUE THREAD

**Issue by ravi247365** [gfranko/jquery.selectBoxIt.js#292]
selectboxit is not accessible for end user using  assistive technology likee  screen reader.

I had raised this issue in one of the accessibility forums , here is the link for that discussion.
http://webaim.org/discussion/mail_message?id=26344
Please follow the above link in order to improve the accessibility  for this plugin.

Thanks 
Ravindra Kumar Jain 
Accessibility Engineer
Onya Digital Sollutons Pvt. Ltd. 
##

**Comment by zkingdesign** [comment]
Any news on this?

**Comment by Sturmkeks** [comment]
Did anyone get this working? I'm trying stuff around by tweaking some code in the plugin js but so far the screen reader doesn't give a crap about the options and doesn't read them on focus. So is there any progress on this?

**Comment by lesliecdub** [comment]
Bumping this. Huge blocker in using selectboxit on production sites. I was able to force accessibility with a ton of extra JS, but IE11 won't allow me to unhide the original <select> for screen readers to see and still properly open the Selectboxit dropdown 👎
